examples/kubernetes-sample-project/README.md

```diff
@@ -0,0 +1,51 @@
+# Kubernetes sample project
+
+This project creates an Nginx deployment using the Pulumi Kotlin Kubernetes SDK.
+
+## How to run
+
+**See the
+[Kubernetes: Installation & Configuration](https://www.pulumi.com/registry/packages/kubernetes/installation-configuration/)
+page in Pulumi Registry for the official documentation.**
+
+
+1. Install Pulumi (see: [docs](https://www.pulumi.com/docs/install/)).
+   ```bash
+   # on MacOS
+   brew install pulumi/tap/pulumi
+   ```
+2. Log in to your Pulumi account (see: [docs](https://www.pulumi.com/docs/cli/commands/pulumi_login/)).
+   ```bash
+   pulumi login
+   ```
+3. Provision a Kubernetes cluster (for test purposes this can be done
+   using [Minikube](https://minikube.sigs.k8s.io/docs/start/)).
+4. Install kubectl (see: [docs](https://kubernetes.io/docs/tasks/tools/)).
+   ```bash
+   # on MacOS
+   brew install kubectl
+   ```
+5. Set up Kubernetes authentication
+   (see: [docs](https://www.pulumi.com/registry/packages/kubernetes/installation-configuration/#setup)). **Note**: when
+   using Minikube, you can use the default configuration and skip this step.
+6. Create a context. **Note**: when using Minikube, you can use the default `minikube` context and skip this step.
+   ```bash
+   kubectl config \
+    set-context <my-context> \
+    --cluster=<my-cluster> \
+    --user=<my-user>
+   ```
```

Comment:
(very minor)

Maybe we could place the note under this point?

<img width="1082" alt="Screenshot 2023-07-17 at 23 18 26" src="https://github.com/VirtuslabRnD/pulumi-kotlin/assets/4415632/03c643a4-a235-4de0-829f-90427c0a86be">


```suggestion
5. Set up Kubernetes authentication
   (see: [docs](https://www.pulumi.com/registry/packages/kubernetes/installation-configuration/#setup)). 
   
   > **Note**
   >
   > When using Minikube, you can use the default configuration and skip this step.

6. Create a context.

   > **Note**
   >
   > When using Minikube, you can use the default `minikube` context and skip this step.
   
   {code}
   
```

----

Unexpected discovery: GitHub Markdown supports callouts. Just in case you didn't know:

`> **Note** GitHub Markdown supports callouts!` 

is turned into:

> **Note** GitHub Markdown supports callouts!


`> **Warning** GitHub Markdown supports callouts, but [some people hate it](https://github.com/orgs/community/discussions/16925#discussioncomment-6470064).` 

is turned into:

> **Warning** GitHub Markdown supports callouts, but [some people hate it](https://github.com/orgs/community/discussions/16925#discussioncomment-6470064).